example_configs/nextcloud_oidc_authelia.md

```diff
@@ -0,0 +1,81 @@
+## Authelia
+
+Set up Authelia according to it's [documentation](https://www.authelia.com/overview/prologue/introduction/), including the [OpenID Connect](https://www.authelia.com/configuration/identity-providers/open-id-connect/) and [Nextcloud instructions](https://www.authelia.com/integration/openid-connect/nextcloud/).
+
+## LLDAP
+
+Set up [LLDAP (Light Lightweight Directory Access Protocol](https://github.com/nitnelave/lldap) according to its standard documentation. 
+
+Add a group and note the name you use. For this tutorial, we're using the group `users`. Create a new user and add it to the `users` group. 
+
+#### Optional:
+Once setup, add an admin or config user and add to the `lldap_strict_readonly` group. This will be the config account used for Nextcloud to read your groups and users from the server. 
```

Comment:
`lldap_strict_readonly` or `lldap_password_manager` ? Can the app modify passwords?

Reply:
Good point, I didn't explicitly note it in the nextcloud ldap settings and it comes unselected here. Especially because of the plain text transport, I'm a bit skeptical about choosing anything besides readonly. As the creator, what do you think?

![Screenshot 2022-11-14 at 2 33 55 PM](https://user-images.githubusercontent.com/44226464/201749648-6441a8a6-9bbf-406d-987c-8017d87ccc7c.png)